player_darwin.h

```diff
@@ -0,0 +1,137 @@
+#include <AudioToolbox/AudioToolbox.h>
+
+//--------------------------------------------
+// AudioPlayer Struct and Associated Functions
+//--------------------------------------------
+
+/*!
+ * AudioPlayer
+ * @field inputUnit - The unit to use to input Audio to the buffer
+ * @field outputUnit - The unit to use to output Audio to the HAL
+ */
```

Comment:
These are doc style comments which are consistent with the way that Apple documents things - personally I want this to be easier to read from the perspective of Mac devs that may want to help - also, many editors will highlight fields/parameters/returns if it's doc commented and will not if it isn't. If you are strongly opposed I can change it.